dotnet-efcore/EntityFrameworkCoreVsDapper/EntityFrameworkCoreVsDapper/Repositories/PersonsRepository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using Dapper;
+using EntityFrameworkCoreVsDapper.EFCore;
+using EntityFrameworkCoreVsDapper.Models;
+using System.Configuration;
+using System.Data.Common;
+using System.Data.SqlClient;
+
+namespace EntityFrameworkCoreVsDapper.Repositories
+{
+    [Orderer(BenchmarkDotNet.Order.SummaryOrderPolicy.FastestToSlowest)]
+    [GroupBenchmarksBy(BenchmarkLogicalGroupRule.ByParams)]
+    public class PersonsRepository
+    {
+        private SqlConnection _dbConnection;
+        private PersonsContext _context;
+
+        public PersonsRepository() 
+        {
+            Setup();
+        }
+
+        [IterationSetup]
+        public void Setup()
+        {
+            _dbConnection = new SqlConnection(ConfigurationManager.ConnectionStrings["PersonsDB"].ConnectionString);
+            _context = new PersonsContext();
+        }
+
+        [Benchmark]
+        [Arguments(10000)]
+        [Arguments(100000)]
+        [Arguments(1000000)]
+        public List<Person> GetXPersonsDapper(int personsNumber)
+        {
+            using (_dbConnection)
```

Comment:
This using needs to be removed as the object is being cleanedup/disposed of in the `Cleanup` method. This also means that we are violating the use after dispose rule as `Cleanup` will be called after this method executes, meaning the call to `_dbConnection.Close()` is executed on a disposed object.